core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -1036,43 +1038,39 @@ public void assertEquals(
 
   private static void assertEqualsAssignUUID(
       MetadataUpdate.AssignUUID expected, MetadataUpdate.AssignUUID actual) {
-    Assert.assertEquals("UUIDs should be equal", expected.uuid(), actual.uuid());
+    assertThat(actual.uuid()).isEqualTo(expected.uuid());
   }
 
   private static void assertEqualsUpgradeFormatVersion(
       MetadataUpdate.UpgradeFormatVersion expected, MetadataUpdate.UpgradeFormatVersion actual) {
-    Assert.assertEquals(
-        "Format version should be equal", expected.formatVersion(), actual.formatVersion());
+    assertThat(actual.formatVersion()).isEqualTo(expected.formatVersion());
```

Comment:
description is missing `as("Format version should be equal")`

Reply:
I didn't think the description is needed because it obvious.

Reply:
I agree that the description doesn't add much value, because if the assertion ever fails, it will be clear enough what the reason of the failure is

Reply:
we don't want to add a description just for the sake of having a description. A description should only exist if we don't have enough contextual information in the assertion. AssertJ typically will provide already enough contextual information if assertions are used appropriately

Reply:
Thanks @nastra for details. Agreed.